symbols.c

```diff
@@ -174,13 +174,93 @@ sort_by (sort_mode_t wanted)
     }
 }
 
+static const struct symbol *
+hint_accumulator (const struct symbol *first, word_t value)
+{
+  const struct symbol *symbol = first;
+
+  /* Look for a single-letter symbol that matches. */
```

Comment:
Not sure, but it might be easier to add an address-then-symbol-length sorting mode, then you could just pick the first one that comes up. (Muddle has 3- and 4-character accumulator names...)